content/2_Bronze/Intro_Sets.mdx

```diff
@@ -939,4 +945,224 @@ make their implementation easier.
   </Quiz.Question>
 </Quiz>
 </CPPSection>
+
+<JavaSection>
+<Quiz>
+  <Quiz.Question>
+    What is the time complexity of insertions, deletions, and searches in a sorted set of size $N$?
+    <Quiz.Answer>
+      $\mathcal{O}(1)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N \log N)$
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $\mathcal{O}(\log N)$
+      <Quiz.Explanation>
+		Correct.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+		```java
+		Set<Integer> s = new TreeSet<>();
+		s.insert(1);
+		s.insert(4);
+		s.insert(5);
+		s.insert(2);
+		System.out.println(s.contains(3));
+		```
+    <Quiz.Answer>
+      `true`
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      `false`
+      <Quiz.Explanation>
+        Correct. $3$ isn't present in `s`.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `null`
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      Error: `NullPointerException`
+      <Quiz.Explanation>
+
+      </Quiz.Explanation>
+    </Quiz.Answer>
+
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following is the correct output of this code?
+		```java
+		Integer[] arr = {4, 1, 2, 1, 3, 2, 3, 1, 3, 1};
```

Comment:
why `Integer[]`
that might confuse some people

Reply:
it doesn't work if it's int because java expects an object array

Reply:
why not
```java
Set<Integer> set = new TreeSet<>(Arrays.asList(4, 1, 2, 1, 3, 2, 3, 1, 3, 1));
```

Reply:
oh wow didn't know you could do that